scripts/deploy.sh

```diff
@@ -20,5 +22,9 @@ if [[ "${NETWORK}" != "anvil" && "${NETWORK}" != "hardhat" && "${NETWORK}" != "m
 	fi
 fi
 
-npx hardhat deploy:hyperdrive --show-stack-traces --network ${NETWORK} --config hardhat.config.${NETWORK}.ts
-npx hardhat deploy:verify --show-stack-traces --network ${NETWORK} --config hardhat.config.${NETWORK}.ts
+config_filename="hardhat.config.${NETWORK}.ts"
+if [[ "${NETWORK}" == "hardhat" ]]; then
+	config_filename="hardhat.config.ts"
```

Comment:
Is this the reason why? Why are we doing this?

Reply:
We want to have the file `hardhat.config.ts` so that it plays nicely with editors trying to understand project structure as well as having the ability to run `npx hardhat` and see what commands are available.

Unfortunately, for the network `hardhat`, the resolved config would be `hardhat.config.hardhat.ts`, which is why I include this accomodation.

Reply:
I think `hardhat.config.hardhat.ts` is fine if it works. It feels pretty consistent with the other config names, but I see why you might not like it.

Reply:
We need the filename `hardhat.config.ts` for `npx hardhat` to run without having to point it to a config file... also pretty sure the hardhat language server vscode integration depends on it being present for some features

Reply:
Right, but we don't need the sepolia stuff in the file, right?